server/aws-lsp-codewhisperer/src/language-server/netTransform/artifactManager.ts

```
@@ -282,6 +285,19 @@ export class ArtifactManager {
             this.logging.log('Cannot find artifacts folder')
             return ''
         }
+
+        const customTransformationPath = path.join(this.solutionRootPath, customTransformationFolderName)
+        try {
+            await fs.promises.access(customTransformationPath)
+            try {
+                this.logging.log(`Adding custom transformation folder to artifact: ${customTransformationPath}`)
+                const artifactCustomTransformationPath = path.join(folderPath, customTransformationFolderName)
+                await fs.promises.cp(customTransformationPath, artifactCustomTransformationPath, { recursive: true })
+            } catch (error) {
+                this.logging.warn(`Failed to copy custom transformation folder: ${error}`)
+            }
+        } catch {}
```

lets add log in this catch too. If access is not present then we can identify that thru this new log.

---

+1. Swallowing exceptions is a code smell. If we truly do not need the exception, we should leave a comment explaining why, or even add a log message explaining the benign behavior (e.g. `"Failed to access resources used for custom transformations. Continuing transformation without customizations. ${e}"`).